### PR TITLE
Fix \transn Boarder and Bump Version to 2.1.1

### LINKED ISSUE #19: \transn breaks vertical spacing inside of a square root
As shown below, when you put a decimal number into a square root, wrapped in `\transn`, it displays incorrectly. E.g.: `\sqrt{\transn{0.04}}`

https://codepen.io/barillax/pen/BvVQBX

Screen Shot 2019-04-01 at 3 22 18 PM

I wonder if the fix for this might be similar to https://github.com/OmarIthawi/arabic-mathjax/pull/16 ?

EDIT: this also happens for `\transt{}{}` inside of a square root, i.e.: `%% \sqrt{{\transt{speed}{السرعة}}} %%`

Screen Shot 2019-04-01 at 4 02 10 PM

### PR DESCRIPTION
**Pull request by shadinaif**
The `\transn` operator is not creating its own reserved space. The effect of that can be seen in operations like:

`\sqrt{\transn{1.234}}`
Selection_004

`\transn{1.234}\\\transn{1.234}`
Selection_003

This PR is intended to fix the issue using the CSS file. The result will be:

`\sqrt{\transn{1.234}}`
Selection_001

`\transn{1.234}\\\transn{1.234}`
Selection_002

Fixes #19

### CODE REVIEW

**Comment by shadinaif** [comment]
@OmarIthawi would you please review this :angel:

**Comment by barillax** [comment]
Nice work, @shadinaif . I have verified locally that these changes fix the issues I was experiencing!

I wonder if we should add these example equations to https://github.com/OmarIthawi/arabic-mathjax/blob/master/testcases/testcases.yml ? I don't know what your policy is on when to add new cases there.

* `\sqrt{\transn{1.234}}`
* `\transn{1.234}\\\transn{1.234}`
* ` \sqrt{{\transt{speed}{السرعة}}`

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "arabic-mathjax",
-  "version": "2.1.0",
+  "version": "2.2.0",
```
Comment (OmarIthawi) [inline package.json]:
```suggestion
  "version": "2.0.1",
```

Probably just a simple bump would do, or whatever https://semver.org/ thinks is ok.

**Review by OmarIthawi — APPROVED** [review]
LGTM. Thanks @shadinaif and @barillax!

**Comment by shadinaif** [comment]
@OmarIthawi Thank you. Yes, this can be considered a patch. I have changed it to 2.1.1
Please review again :)

**Review by OmarIthawi — APPROVED** [review]

**Comment by barillax** [comment]
@shadinaif @OmarIthawi Thank you! Could you `npm publish` the new release so we can start using it?

**Comment by shadinaif** [comment]
Thank you @OmarIthawi 

The npm package has been published @barillax :+1:

**Pull request merged**